src/Core/Components/InfoButton/FluentInfoButton.razor

```diff
@@ -0,0 +1,20 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@inherits FluentComponentBase
```

Comment:
Using this base class, you need to use all properties in your razor file: `Id`, `Class`, `Style` and `AdditionalAttributes`

Reply:
Id I had already used it, but what do I use Class and Style for?
AdditionalAttributes even if I don't associate it, it's always available, right?

Reply:
`Class` and `Style` you use in 
```
/// <summary />
protected override string? ClassValue => new CssBuilder(base.ClassValue)
    .AddClass("fluent-rating")
    .Build();

/// <summary />
protected override string? StyleValue => new StyleBuilder(base.StyleValue).Build();
```

and then output these in component so someone can always add their own class and style value

Reply:
ok, but where do I use them?

Reply:
Ah. You could pass them through to the `FluentTooltip`. Probably better to not us the builders then